thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datasource/ThirdEyeCacheRegistry.java

```diff
@@ -88,7 +88,7 @@ private static void initDataSources(ThirdEyeConfiguration thirdeyeConfig) {
       }
       // Query Cache
       Map<String, ThirdEyeDataSource> thirdEyeDataSourcesMap = DataSourcesLoader.getDataSourceMap(dataSources);
-      QueryCache queryCache = new QueryCache(thirdEyeDataSourcesMap, Executors.newFixedThreadPool(10));
+      QueryCache queryCache = new QueryCache(thirdEyeDataSourcesMap, Executors.newCachedThreadPool());
```

Comment:
another star of the show. please add a comment to the PR summary

Reply:
No problem